From foo@baz Tue Aug 25 16:04:46 PDT 2009
Date: Tue, 25 Aug 2009 16:04:46 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: cowloop: add TODO file

Add a TODO file with a few things that needs to be fixed up.

Cc: "H.J. Thomassen" <hjt@ATComputing.nl>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/cowloop/TODO |   11 +++++++++++
 1 file changed, 11 insertions(+)

--- /dev/null
+++ b/drivers/staging/cowloop/TODO
@@ -0,0 +1,11 @@
+TODO:
+	- checkpatch.pl cleanups
+	- run sparse to ensure clean
+	- fix up 32/64bit ioctl issues
+	- move proc file usage to debugfs
+	- audit ioctls
+	- add documentation
+	- get linux-fsdevel to review it
+
+Please send patches to "H.J. Thomassen" <hjt@ATComputing.nl> and
+Greg Kroah-Hartman <gregkh@suse.de>
